jms/serializer-bundle/2.0/etc/packages/prod/jms_serializer.yaml

```diff
@@ -0,0 +1,4 @@
+jms_serializer:
+    visitors:
+        xml:
+            format_output: false
```

Comment:
couldn't you do it in the bundle directly, with a smart default (i.e. using `%kernel.debug%` as default value for this setting) ?